wordpress-mobile/WordPress-iOS
# Posts & Pages: Hide author if "me" is selected or blog has only one author

Part of https://github.com/wordpress-mobile/WordPress-iOS/issues/21900

## Description
- Hides the author badge if "me" is selected, or if the blog has only one author
- Note that the search screen always shows the author

## How to test
- Switch to a site that only has one author
- Go to the posts screen
- Verify no author badge is shown
- Switch to a site that has multiple authors
- Go to the posts screen and show posts from "me"
- Verify no author badge is shown
- Show posts from "everyone"
- Verify the author badge is shown


https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/8b848f91-f492-4599-bd1b-db87ec0cbb26



## Regression Notes
1. Potential unintended areas of impact
Post screen / search screen

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested manually

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:


UI Changes testing checklist:
